### ISSUE TITLE
R help doesn't work in R terminal

### ISSUE THREAD

**Issue by royfrancis** [rocker-org/devcontainer-images#69]
When asking for help from the R terminal using `?`, I get the following error: 'Couldn't show help for path:'.

![Screenshot 2024-07-10 at 14 44 52](https://github.com/rocker-org/devcontainer-images/assets/5083500/e86d2ffd-d242-48fb-83f0-727521fe1716)

Details:

I am running a docker container ([ghcr.io/rocker-org/devcontainer/geospatial:4.4](https://github.com/rocker-org/devcontainer-images/pkgs/container/devcontainer%2Fgeospatial)) on a Mac M3. And I have attached to the running container through VS Code Dev Containers extension. I get this error in the R terminal (Maybe Radian?). If I open up a regular terminal and enter R and ask for help, it opens in the terminal itself.

![Screenshot 2024-07-10 at 14 47 28](https://github.com/rocker-org/devcontainer-images/assets/5083500/543fa3f4-b3fd-4220-9a03-3693d7fea144)

Not really sure if this is related to the container or VS Code or an extension. Anyone else experiencing this? Happy to give more info if needed. 

```
Docker Desktop: 4.30.0
VS Code: 1.91.0
Extensions:
  REditorSupport: 2.8.4
  Dev Containers: 0.375.1
```

**Comment by eitsupi** [comment]
I have been using these images for the past several years on Windows and Linux and have never experienced such a situation.

Could it be due to the amd64 emulation of the arm mac? Does the same thing happen when using the arm64 images (such as `ghcr.io/rocker-org/devcontainer/r-ver:4.4`)?
In any case I am unable to use macOS so debugging is not possible.

**Comment by royfrancis** [comment]
I have always used `--platform linux/amd64` to avoid the apple silicon related issues, so i decided to try aarch64 as you suggested.

```
docker pull ghcr.io/rocker-org/devcontainer/r-ver:4.4
docker run -ti --rm ghcr.io/rocker-org/devcontainer/r-ver:4.4 bash
```

I see the same.

![Screenshot 2024-07-11 at 10 03 29](https://github.com/rocker-org/devcontainer-images/assets/5083500/d5f2c322-8173-43df-8575-3d5e6ab12733)

The web based help is unable to open. If it works for you, then it cannot be an issue with the container. It's probably something to do with my local setup. I am not exactly sure how it's supposed to open. In the viewer? Or somewhere else? Is it a port issue? 

@eitsupi Would you be able to share a screenshot of how it looks like for you?

**Comment by eitsupi** [comment]
@royfrancis If my guess is correct, you have misunderstood the use of VS Code Dev Container. Please check the documentation.
<https://code.visualstudio.com/docs/devcontainers/create-dev-container>

You should be able to check with GitHub Codespaces for the correct behavior (I can share screenshots right now).
<https://github.com/rocker-org/devcontainer-try-r>

**Comment by royfrancis** [comment]
Hmmm.. The examples you have linked uses `devcontainer.json` to set up some config, install extensions etc.. I just didn't use that for my example. I am not sure if that should make a difference with regard to the issue with R help.

Anyway, I also tried out the Github Codespaces link that you provided. Same result when I ran it in my local installation of VS Code.

![Screenshot 2024-07-11 at 20 30 38](https://github.com/rocker-org/devcontainer-images/assets/5083500/474a368c-a93f-4bac-94af-23b015e72451)

But it worked when I ran it in the browser.

![Screenshot 2024-07-11 at 20 37 22](https://github.com/rocker-org/devcontainer-images/assets/5083500/c9d95513-d951-4dc7-8a86-c144a50712f8)

So, perhaps it's related to my VS Code installation? Hmmm 🤔
